client/components/card/index.jsx

```diff
@@ -40,7 +40,7 @@ export default React.createClass( {
 
 		return React.createElement(
 			element,
-			assign( {}, this.props, { className } ),
+			assign( {}, omit( this.props, 'compact' ), { className } ),
```

Comment:
> Instead of omitting, wouldn’t it be more future-proof to use `pick`?

It would be, but I don't think it's really possible here, since through the [`tagName`](https://github.com/Automattic/wp-calypso/pull/7407/files#diff-2f2d5887ccf786d867968edbd0725397R32) prop, `element` can be pretty much any DOM element at all. And we wouldn't want to whitelist all possible attributes for all of them :-)